testsuite/ext-tools/maintenance_json_generator.py

```diff
@@ -0,0 +1,193 @@
+import argparse
+import json
+import subprocess
+import sys
+
+import requests
+
+# Default dictionary for client tools, essentially anything that is common between SUMA versions
+defaultdict = {
+    "sle12sp4_client": "/SUSE_Updates_SLE-Manager-Tools_12_x86_64/",
+    "sle12sp4_minion": "/SUSE_Updates_SLE-Manager-Tools_12_x86_64/",
+    "sle12sp5_client": "/SUSE_Updates_SLE-Manager-Tools_12_x86_64/",
+    "sle12sp5_minion": "/SUSE_Updates_SLE-Manager-Tools_12_x86_64/",
+    "sle15_client": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                     "/SUSE_Updates_SLE-Product-SLES_15-LTSS_x86_64/"],
+    "sle15_minion": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                     "/SUSE_Updates_SLE-Product-SLES_15-LTSS_x86_64/"],
+    "sle15sp1_client": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"],
+    "sle15sp1_minion": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Product-SLES_15-SP1-LTSS_x86_64/"],
+    "sle15sp2_client": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Product-SLES_15-SP2-LTSS_x86_64/"],
+    "sle15sp2_minion": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Product-SLES_15-SP2-LTSS_x86_64/"],
+    "sle15sp3_client": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Basesystem_15-SP3_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Server-Applications_15-SP3_x86_64/"],
+    "sle15sp3_minion": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Basesystem_15-SP3_x86_64/",
+                        "/SUSE_Updates_SLE-Module-Server-Applications_15-SP3_x86_64/"],
+    "sle15sp4_minion": ["/SUSE_Updates_SLE-Manager-Tools_15_x86_64/",
```

Comment:
15.3 client please

Reply:
I think you meant 15.4 client, which is now added